#!/usr/bin/python3


def raise_exception_msg(message="Python is fun"):
    raise NameError(message)
